lib/tower_cli/resources/job.py

```diff
@@ -89,6 +90,8 @@ def launch(self, job_template=None, tags=None, monitor=False, timeout=None,
         # Get the job template from Ansible Tower.
         # This is used as the baseline for starting the job.
 
+        tags = kwargs.get('tags', None)
```

Comment:
sure, and for right here, the only reason this line about tags was added was because it was absorbed into kwargs. That's going in the right direction, so we'll leave it for now. I'm still gearing up to run some test cases with the feature in general right now.